chainservice/chainservice.go

```diff
@@ -631,8 +630,8 @@ func (cs *ChainService) ActionPool() actpool.ActPool {
 }
 
 // APIServer returns the API server
-func (cs *ChainService) APIServer() *api.Server {
-	return cs.api
+func (cs *ChainService) APIServer() *api.CoreService {
+	return cs.api.CoreService()
```

Comment:
i think just return `cs.api` here, so no need to change `coreService` to `CoreService`